packages/components/label/src/label.tsx

```diff
@@ -0,0 +1,79 @@
+import type { MessageDescriptor } from 'react-intl';
+
+import React from 'react';
+import invariant from 'tiny-invariant';
+import Text from '@commercetools-uikit/text';
+import RequiredIndicator from './required-indicator';
+
+type TBasicTextProps = {
+  intlMessage?: MessageDescriptor;
+  children?: React.ReactNode;
+};
+type TLabelProps = {
+  /**
+   * Use this to set the relationships between the label and a non-form element.
+   * For example:
+   *   <label id="foo">
+   *   <div aria-labelledby="foo">
+   */
+  id?: string;
+  /**
+   * Use this to set the relationships between the label and a form element.
+   * For example:
+   *   <label for="foo">
+   *   <input id="foo">
+   *
+   *   <label id="foo">
+   *   <input aria-labelledby="foo">
+   */
+  htmlFor?: string;
+  isBold?: boolean;
+  isRequiredIndicatorVisible?: boolean;
+  tone?: 'primary' | 'inverted';
+} & TBasicTextProps;
+
+const Label = (props: TLabelProps) => {
+  if (props.id && props.htmlFor) {
+    invariant(
+      false,
```

Comment:
Isn't this more of how the inventors thought of it? :)

```suggestion
    invariant(
      props.id && props.htmlFor,
```

Reply:
haha I just migrated blindy. good catch, bud  👋 (slap)

Reply:
actually the boolean checked passed in is incorrect